src/hotspot/cpu/x86/x86.ad

```diff
@@ -2834,7 +2851,9 @@ public:
     Flag_clears_zero_flag     = Node::_last_flag << 9,
     Flag_clears_overflow_flag = Node::_last_flag << 10,
     Flag_clears_sign_flag     = Node::_last_flag << 11,
-    _last_flag                = Flag_clears_sign_flag
+    Flag_ndd_demotable_flag   = Node::_last_flag << 12,
+    Flag_ndd_commutative_flag = Node::_last_flag << 13,
```

Comment:
Why are these called Flag_ndd_demotable_flag and Flag_ndd_commutative_flag instead of just Flag_ndd_demotable and Flag_ndd_commutative? Would make more sense to me.
The flags above end in flag cause they mean that the instruction does set/clear the flag in the EFLAGS register of the CPU